src/index.d.ts

```diff
@@ -3155,6 +3155,40 @@ declare namespace esb {
         docType?: string
     ): PercolateQuery;
 
+    /**
```

Comment:
Could you keep the doc comment similar to the source code here? Apologies for the redundant work 🙏🏼

Reply:
No worries! Now both have the same doc